### PR TITLE
Development

### PR DESCRIPTION
**Pull request by raquel-ucl**
Flask app PoC to show work on ES to Eleanor and Steve.

Needs quite a lot of refactoring but it works.

### CODE REVIEW

File: query_glossary_word.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import sys
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search, Q
+
+client = Elasticsearch()
+
+def search(fieldname, word):
```
Comment (ageorgou) [inline query_glossary_word.py]: I guess we don't need this and can use the things in `search.search` instead?

File: search-app/search_app/search/search.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                                    **{"entries.{}".format(fieldname): word}))
+        results = search.execute()
+        return results
+
+    def _get_results(self, results):
+        '''
+        Compile list of results.
+        '''
+        result_list = []
+        for glossary_results in results.hits.hits:
```
Comment (ageorgou) [inline search-app/search_app/search/search.py]: I haven't managed to find a way to make this part shorter :( You can do `for r in results`, but then you don't have access to the `inner_hits` (which is what we want), just the whole document.

**Comment by ageorgou** [comment]
I found a bug which gave fewer results than we should be getting: we were only checking the part of the results corresponding to a single indexed document. I don't remember if we did this for debugging at first, but it now seems to give the expected results (for the few things I compared it against the sux glossary on the ORACC website).

**Comment by raquel-ucl** [comment]
Copied mostly into this other repo: https://github.com/oracc/oracc-rest

**Pull request closed without merging**